docs/quickstart/bootstrap.rst

```diff
@@ -11,8 +11,8 @@ Mount this ISO file at the specific following path:
 
 .. code-block:: shell
 
-   root@bootstrap $ mkdir -p /srv/scality/metalk8s-|version|
```

Comment:
Why are you hardcoding stuff? Now it will be wrong for 2.1 and 2.2…

Reply:
Because `|version|` is not render in code-block that not working before this PR

Reply:
Then we should probably find an alternative.

Reply:
Replace the `code-block` directive with `parsed-literal`, should work (though lost highlighting).

Reply:
Nice, I will do that thanks